yaksh/bash_files/sample2.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+cat $1 | cut -d: -f2 | paste -d: $3 - $2
```

Comment:
This sample script is used in evaluator to check user answer. 
This script acts as test case for bash question.
It is not related to file uploads.
same for `sample2.args`